Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -107,14 +107,27 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
     }
     Add-AnalyzedResultInformation @params
 
-    $displayValue = $credentialGuardValue = $osInformation.RegistryValues.CredentialGuard -ne 0
+    $credGuardRunning = $false
+    $credGuardUnknown = $osInformation.CredentialGuardCimInstance -eq "Unknown"
+
+    if (-not ($credGuardUnknown)) {
+        # CredentialGuardCimInstance is an array type and not sure if we can have multiple here, so just going to loop thru and handle it this way.
+        $credGuardRunning = $null -ne ($osInformation.CredentialGuardCimInstance | Where-Object { $_ -ne 0 })
+    }
+
+    $displayValue = $credentialGuardValue = $osInformation.RegistryValues.CredentialGuard -ne 0 -or $credGuardRunning
```

Comment:
We keep the `$osInformation.RegistryValues.CredentialGuard -ne 0` for backward compatibility, right?

Reply:
Hm, looks like it can also be configured via GPO. We might need to validate this registry path as well: `SOFTWARE\Policies\Microsoft\Windows\DeviceGuard`

```
ValueName: lsacfgflags
item: decimal: 0 => Disabled
item: decimal: 1 => Enabled with UEFI lock
item: decimal: 2 => Enabled without lock
item: decimal: 3 => Not Configured
```

https://gpsearch.azurewebsites.net/#10939

Reply:
> We keep the `$osInformation.RegistryValues.CredentialGuard -ne 0` for backward compatibility, right?
> 
> **_Edit:_** Looks like this is still a supported way to configure Credential Guard (as per: https://learn.microsoft.com/en-us/windows/security/identity-protection/credential-guard/configure?tabs=intune#configure-credential-guard-with-intune)

yes, I wanted to keep it there just in case on older systems doesn't have the option of `Get-CimInstance` for credential guard.